dll/win32/comctl32/draglist.c

```diff
@@ -155,7 +155,7 @@ DragList_SubclassWindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam,
                     SetCursor(data->cursor);
                     break;
                 case DL_COPYCURSOR:
-                    data->cursor = LoadCursorW(COMCTL32_hModule, (LPCWSTR)IDC_COPY);
+                    data->cursor = LoadCursorW(COMCTL32_hModule, MAKEINTRESOURCEW(IDC_COPY));
```

Comment:
Isn't it winesync'ed?

Reply:
MAKEINTRESOURCEW is just a cast to LPCWSTR anyway so I would perhaps just keep the cast instead of having to #ifdef __REACTOS__ in wine synced files (IMHO).

Reply:
No, it's `LPWSTR` instead, and with `ULONG_PTR` additional cast, see https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/winuser.h;hb=7e89227a005f3c60ddd1f93dcb797e5583ef857d#l582. So these are a bit different things and then having a diff makes sense.

Reply:
> No, it's `LPWSTR` instead

Like that matters in the slightest. The ULONG_PTR intermediate cast is to avoid a size warning so the question becomes, does the current wine code cause a warning for 64-bit builds?

Reply:
> does the current wine code cause a warning for 64-bit builds?

At least in draglist.c file, in that part the code is absolutely identical to current Wine.
I checked: both GCC and MSVC amd64 0x0502 logs have no any warnings at all from this draglist.c file.
So I still doubt: is it better to keep these changes, but in ROS diff, or remove them at all? :thinking:

Reply:
Moved into `#ifdef __REACTOS__` preprocessor.

Reply:
Added the change into ifdef in comctl32/toolbar.c part too.